support/rally.js

```diff
@@ -176,6 +208,12 @@ export default class Rally {
       return;
     }
 
+    // When paused, log an error and do not send data.
+    if (this._state === "paused") {
+      console.error("Rally.sendPing - Study is currently paused, not sending data");
```

Comment:
Do we really need this to be an `error`? This is not really actionable by users nor developers, or am I missing something?

Reply:
Hm, I would want people who are testing (or normal users) to be able to report it to us. I think `warn` at least is appropriate, but I'm OK with `debug` here .

Reply:
IIRC Firefox shows `debug` by default in the console but Chrome does not.

On a different note, I am starting to think maybe this check should be in the core add-on code and we shouldn't just expect `rally.js` to do the right thing, that feels safer in the face of a misbehaving add-on.

Reply:
Oh yeah, indeed. See what I'm doing in #352 , because I'm doing the same :)